vars/runTestSh.groovy

```diff
@@ -0,0 +1,55 @@
+// vars/runTestSh.groovy
```

Comment:
This name is not really very descriptive of what this function is.

How about `runTestUnitTest` instead?